test/plot-schema.json

```diff
@@ -104,6 +104,21 @@
   }
  },
  "config": {
+  "accessibility": {
+   "description": "Accessibility options: which library to use; whether to enable, options to pass to the library, info to pass to the library, closedCaptions to control how plotly renders the closed-captions element. chart2music is supported and options here include Options and Info from https://www.chart2music.com/docs/API/Config. ",
+   "dflt": {
+    "closedCaptions": {
+     "elClassname": "c2m-plotly-closed_captions",
```

Comment:
We might want to discuss whether we want to use the name of the tool in the schema.